### PR TITLE
Validation fixes for running with evolving CISM

### PR DESCRIPTION
**Pull request by mvertens**
### Description of changes
Validation fixes for running with evolving CISM

### Specific notes
1. CMEPS changes that were introduced as part of the validation of the compset  I1850Clm50SpG -which contains an evolving ice sheet. 
2. This work also needed to be accompanied by associated CISM changes that are in the nuopc_dev hash listed below.
3. Also validated using the CTSM master code base. As a result all baseline comparisons that contain CLM will fail.
4. A new reworked testlist_drv.xml that contains a test fo rI1850CLM50SpG and also removes all SMS tests that had duplicate ERS tests.

Contributors other than yourself, if any: @billsacks 

CMEPS Issues Fixes: #128

Are changes expected to change answers?
 - [x] bit for bit (all compsets not containing CTSM)
 - [ ] different at roundoff level
 - [x] more substantial (all compsets containing CTSM)

Any User Interface Changes (namelist or namelist defaults changes)?
 - [ ] Yes
 - [x] No

Testing performed if application target is CESM:(either UFS-S2S or CESM testing is required):
- [ ] (required) CIME_DRIVER=nuopc scripts_regression_tests.py
   - machines:
   - details (e.g. failed tests):
- [x] (required) CESM testlist_drv.xml
   - machines and compilers: cheyenne intel
   - details (e.g. failed tests): two sets of baselines were created
        cmeps_baselines/nov24 contains output for the original testlist_drv.xml
        cmeps_baselines/nov24newtests contains output for the new testlist_drv.xml and should be used for future comparisons
- [ ] (optional) CESM prealpha test
   - machines and compilers
   - details (e.g. failed tests):

Testing performed if application target is UFS-S2S:
- [ ] (required) UFS-S2S testing
   - description:
   - details (e.g. failed tests):

Hashes used for testing:
- [x] CESM:
  - repository to check out: https://github.com/ESCOMP/CESM.git
  - branch: nuopc_dev
  - hash: 23cb395
- [ ] UFS-S2S, then umbrella repostiory to check out and associated hash:
  - repository to check out:
  - branch:
  - hash:

### CODE REVIEW

File: mediator/med_phases_prep_glc_mod.F90 [hunk 1]
```diff
@@ -442,126 +452,96 @@ subroutine med_phases_prep_glc_avg(gcomp, rc)
...
+       do n = 1, size(fldnames_fr_lnd)
+          call ESMF_FieldBundleGet(FBlndAccum_lnd, fieldname=fldnames_fr_lnd(n), field=lfield, rc=rc)
           if (chkerr(rc,__LINE__,u_FILE_u)) return
-          call ESMF_FieldGet(lfield, farrayptr=data2d_import, rc=rc)
+          call ESMF_FieldGet(lfield, farrayptr=data2d, rc=rc)
           if (chkerr(rc,__LINE__,u_FILE_u)) return
-          data2d(:,:) = data2d_import(:,:)
+          if (FBlndAccumCnt > 0) then
+             ! If accumulation count is greater than 0, do the averaging
+             data2d(:,:) = data2d(:,:) / real(FBlndAccumCnt)
```
Comment (jedwards4b) [inline mediator/med_phases_prep_glc_mod.F90:506]: should be real(FBlndAccumCnt, kind=r8)

**Review by jedwards4b — APPROVED** [review]
One minor nit.

File: mediator/med_phases_prep_glc_mod.F90 (hunk 1 above)
Comment (mvertens) [reply mediator/med_phases_prep_glc_mod.F90:506]: Thanks for catching this. Since this might change answers - can we please add it in an upcoming PR?

**Review by DeniseWorthen — APPROVED** [review]
